### PR TITLE
feat: Update Points

### LINKED ISSUE #16: Update Scores
Update score variable of the leaderboard model. 
You may use variables for values to do the same instead of hardcoding it if the schema isn't clear yet, such that we will be able to simply update values of those variables later.

### PR DESCRIPTION
**Pull request by anshsrtv**
### Fixes #16
Using the `.env`, we can now change the points whenever the weightage is decided. :v:

### CODE REVIEW

File: profiles/views.py [hunk 1]
```diff
@@ -96,19 +101,21 @@ def issue(request):
...
                     leaderboard.milestone_achieved = True
-                leaderboard.save()
             elif label["name"] == 'hard':
                 leaderboard.hard_issues_solved += 1
-                leaderboard.save()
+                leaderboard.points += hard_issue_points
             else:
                 pass
+            
+            leaderboard.save()
```
Comment (priyanshisharma) [inline profiles/views.py:118]: By indentation I suppose this is inside the `for label in labels:` loop, but leaderboard doesn't need to be saved for every label, but only when there is a change in values. Please crosscheck this.

File: README.md [hunk 2]
```diff
@@ -8,13 +8,22 @@
     1. windows - Download installer from https://www.python.org/downloads/release/python-370/.
 1. Activate the virutal environment: `source env/bin/activate`
 1. Install all the dependencies in `requirements.txt` file: `pip install -r requirements.txt`
+1. Make a copy of `.env.sample` to `.env` and change the values of variables with original values. [Click here ](#example-env-file) for sample .env file.
```
Comment (priyanshisharma) [inline README.md:11]: We need to add the `env.sample` file too.

File: profiles/views.py (hunk 1 above)
Comment (anshsrtv) [reply profiles/views.py:118]: This line was being repeated in most of the conditions, that is why I added it here. I will revert the changes. :disappointed:

File: README.md (hunk 2 above)
Comment (anshsrtv) [reply README.md:11]: Added Roger Copy :joy:

File: profiles/views.py (hunk 1 above)
Comment (priyanshisharma) [reply profiles/views.py:118]: 😅😓

**Pull request merged**